DTS-STN/oas-unblock-bot
# Feature/ll adaptivecards

# Description

Adding in Microsoft [AdaptiveCards](https://adaptivecards.io/explorer/) framework to solve design problems like longer text strings, adding links, creating bullet points, and embedding images.

## Test Instructions

1. Start bot
2. Select 'yes' from unblock prompt
3. Observe the text and image formats in the following text prompts
4. Complete user input of direct deposit info as prompted
-----
5. Open code editor and change src/dialogs/unblockDialogs/unblockDirectDeposit.ts 
const CARD_DEBUG = null; to = true;
6. Restart the Bot, see the alternate approach to the same flow

## Help Requested

- Unit testing framework

## Checklist
